Cargo.toml

```
@@ -114,6 +114,9 @@ aes = "0.8"
 fpe = "0.6"
 zip32 = "0.1"
 
+# Runtime
+tokio = { version = "1.21.0", features = ["rt-multi-thread"] }
```

We cannot enable this flag at a library level; it needs to be up to the application.
```suggestion
tokio = "1.21"
```

---

I have moved the tokio dependency to zcash_client_backend an optional and behind a "sync" feature for BlockCache. rt-multi-thread is only a dev-dependency now for the doc-test